docs/synthesis.md

```diff
@@ -146,12 +146,35 @@ Also, if present in the telemetry, these attributes are also added to the entity
 
 [guid_spec]: guid_spec.md
 
+#### Prefixed tags
+
+This is a special section where specify prefixes.
```

Comment:
Not sure what is this trying to state but without this phrase i feel it reads just as fine